### ISSUE TITLE
"Continue" doesn't work for multiple breakpoints in the same script

### ISSUE THREAD

**Issue by olestourko** [godotengine/godot-vscode-plugin#296]
OS: Ubuntu 20.04
Godot: 3.3.3
VSCode: 1.59.1

When setting two breakpoints within the same script, only the first one gets hit. When I press "continue" the next one doesn't get triggered. Continue **does** work correctly for breakpoints in separate scripts.

![Screenshot_2021-09-01_18-14-37](https://user-images.githubusercontent.com/3958783/131752518-02b447d4-6df8-4480-8b10-6868d090aa81.png)

^ Only the one on L12 works. L14 not triggered on Continue. L14 does work if I remove the L12 breakpoint.

**Comment by olestourko** [comment]
My `launch.json` file:

```
{
    // Use IntelliSense to learn about possible attributes.
    // Hover to view descriptions of existing attributes.
    // For more information, visit: https://go.microsoft.com/fwlink/?linkid=830387
    "version": "0.2.0",
    "configurations": [
        {
            "name": "GDScript Godot",
            "type": "godot",
            "request": "launch",
            "project": "${workspaceFolder}",
            "port": 6007,
            "address": "127.0.0.1",
            "launch_game_instance": true,
            "launch_scene": false
        }
    ]
}
```

**Comment by gabstv** [comment]
It's happening to me too.  The debugger just gives up mid file.

**Comment by raulsntos** [comment]
Closed by #324
